_posts/2022-11-07-relicensing.md

```diff
@@ -0,0 +1,201 @@
+we do not want to dictate how our developers and users should license
+their code, but we also wish to ensure that work on the MDAnalysis
+library remains open and free.
+
+Doing so would ultimately benefit the MDAnalysis community, increasing
+the number of codes which can use MDAnalysis, and enabling users in
+corporate environments to use the library with more certainty. The
+reduced licensing complexity also paves the way for our proposed
+[MDAKit ecosystem]({{ site.baseurl }}{% post_url 2022-08-24-mdakits-intro %}).
+
+
+### Why now the LGPL v3+?
+
+We therefore propose to undergo the process of **relicensing MDAnalysis
+under the [Lesser GNU Public License v3 (or any later versions)][LGPL]**.
+This open source license fulfills a number of important requirements for us:
+
+1. Downstream codes are able to freely import or link to MDAnalysis
+   library components without impacting the license choice of the
+   downstream code.
+   
+2. Downstream codes are able to use and subclass any MDAnalysis components
+   under its application programming interface (classes, methods, and
+   data objects), without impacting the license choice of the
+   downstream code.
+   
+3. Codes which either copy or extend the MDAnalysis library should
+   fall under the copyleft license requirements of the MDAnalysis
+   library license.
+   
+Thus, it is our view that the LGPL v3+ license gives people the freedom
+to choose any license for their own code that *makes use* of the MDAnalysis
+library as a whole (namely ``import MDAnalysis`` or subclassing). This
+includes closed / commercial licenses (although we encourage the use of
+open source licenses). However, one would not be able to just take parts of
+the MDAnalysis code and put it into another piece of code unless the
+other code is then *also* licensed under a compatible copyleft license
+(e.g. GPLv3+/LGPLv3+).
+
+We considered other popular licenses but none fulfilled our requirements.
+
+   
+## How will the relicensing process work?
+
+As of writing, MDAnalysis has [over 160 contributors][contributors].
+All of which have contributed code under the terms of the GPL v2+
+license. We also have a large user community which uses the library
+for many wonderful scientific applications, including several
+downstream libraries.
+
+Ultimately, the final decision on relicensing [rests with code
+authors](#Contacting-contributors).  However, we fully recognise that
+this is a big change for the MDAnalysis user base and the wider
+molecular sciences community. We are, as always, fully invested in
+ensuring that our actions reflect the needs of our community. We
+therefore want to give everyone an opportunity to [ask questions about
+or comment on the relicensing effort](#Consultation-period) as part of
+this process.
+
+
+### Consultation period (7th November until 5th December 2022)
+
+We will start the process with a 28 days open consultation period.
```

Comment:
```suggestion
We will start the process with an open consultation period lasting 28 days from 7th November to 5th December 2022 (anywhere on earth).
```

I assume we don't be strict wrt timezones.

Reply:
Honestly if someone posts on the last day that doesn't help much with the conversation. I'd rather not have "anywhere on earth" here, it's not a deadline for them, it's one for us.